# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
# Feedback definition
# The feedback specifies how many goals have already been chosen
bool status

